src/components/auth/RegistrationForm.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <form
+    class="registration-form"
+    @submit.prevent="register(user)"
+  >
+    <InputCustom
+      label="username"
+      v-model="user.username"
+      :required="true"
+      @input="clearMessage"
+    />
+    <InputCustom
+      label="e-mail"
+      type="email"
+      v-model="user.email"
+      :required="true"
```

Comment:
as I remember, if this line would look like this "required" it's gonna be the same according to the vue functionality